DOC.md

```diff
@@ -1128,8 +1128,9 @@ ls.add_snippets("all", {
   * `dedent`: remove indent common to all lines in `format`. Again, makes
   	passing multiline-strings a bit nicer (default true).
 
-There is also `require("luasnip.extras.fmt").fmta`. This only differs from `fmt`
-by using angle-brackets (`<>`) as the default-delimiter.
+Also `require("luasnip.extras.fmt").fmta` and `require("luasnip.extras.fmt").fmts`. This is
```

Comment:
Looks like the first sentence is missing some words? :sweat_smile:
(and a small mishap with `` ` `` in the line below)

Reply:
Huh? Sorry, this is a boring mistake.

Reply:
I was totally unaware of it. Thank you very much.